engine/flink/executor/src/test/scala/pl/touk/nussknacker/engine/process/runner/FlinkStreamingProcessMainSpec.scala

```diff
@@ -29,7 +29,7 @@ class FlinkStreamingProcessMainSpec extends FlatSpec with Matchers with Inside {
         .emptySink("out", "monitor")
 
     FlinkTestConfiguration.setQueryableStatePortRangesBySystemProperties()
-    FlinkStreamingProcessMain.main(Array(ScenarioParser.toGraphProcess(process).marshalled,
+    FlinkStreamingProcessMain.main(Array(ScenarioParser.toJson(process).spaces2,
```

Comment:
Can we move converting canonical to json string to CanonicalProcess?

Reply:
`ScenarioParser` convert not canocnialprocess but espprocess produced by DSL. But at next change I'll try to add transition methods for espprocess -> canonicalprocess  which will reduce usage of this parsers, marshallers and canonizers